runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/bot.json

```diff
@@ -20,6 +20,9 @@
     "Bot.Collection.Dialog.Modify.Form.PublicBots.Label": "Public bots included:",
     "Bot.Collection.Dialog.Modify.Form.Users.Label": "Users",
     "Bot.Collection.Dialog.Modify.Form.ErrorMessage": "Something went wrong!",
+    "Bot.Collection.Dialog.Modify.Form.ErrorMessage.noname":"Name cannot be null",
```

Comment:
We don't use such names like `null` towards end-users. Put it simply, `Name cannot be empty` or `This field is required` choose your way

Reply:
bump